asana/asana_test.go

```diff
@@ -14,6 +15,7 @@ var (
 	client *Client
 	mux    *http.ServeMux
 	server *httptest.Server
+	ctx    = context.Background()
```

Comment:
I prefer to write 3 letters instead of context.Background() in tests, but it's not a big deal. I remove this global variable and put context.Background().